superduperdb/backends/query_dataset.py

```diff
@@ -201,7 +214,10 @@ def __getitem__(self, index):
                     s[k] = r[k]
         else:
             s = r
-        return self.transform(s)
+        out = self.transform(s)
```

Comment:
Maybe easiest to explain with an example?

Reply:
Same as above